### DISCUSSION TITLE
Προθεσμίες για ασκήσεις Mπόνους

### DISCUSSION THREAD

**Discussion by p14kairas** [courses-ionio/hci#1655] [Announcements]
Εάν έχουμε χάσει την προθεσμία της 6ης εβδομάδας θα μπορούσαμε να ασχοληθούμε με το προαιρετικό μπόνους έστω και για μικρότερη βαθμολόγια και εάν ναι τι ποσοστό  του βαθμού θα χάσουμε?

**Comment by epidrome** [comment]
Τα μπόνους είναι συνήθως εργασίες μεγάλων απαιτήσεων που απευθύνονται σε όσους θέλουν να μάθουν περισσότερα για το αντίστοιχο θέμα. Επομένως, η βαθμολογία τους δεν έχει αναλογία με την βαθμολογία των ασκήσεων του μαθήματος και οι προθεσμίες πρέπει να τηρηθούν απόλυτα ώστε να είναι εφικτή η ολοκλήρωση τους.
